spec/requests/as_server_app/comment_thread_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'yt/models/comment_thread'
+
+describe Yt::CommentThread, :server_app do
+  subject(:comment_thread) { Yt::CommentThread.new attrs }
+
+  context 'given an existing comment thread ID about a channel' do
+    let(:attrs) { {id: 'z13vsnnbwtv4sbnug232erczcmi3wzaug'} }
+
+    it { expect(comment_thread.video_id).to be_nil }
+    # Not sure why but channel_id seems always missing value
```

Comment:
Typical of YouTube to have wrong documentation.

Since you probably don't need this for your app, just remove it completely from code and tests.